libmscore/sym.cpp

```diff
@@ -2969,8 +2969,10 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       "Bakiye (sharp)",
       "Accidental bracket, left",
       "Accidental bracket, right",
-      QT_TRANSLATE_NOOP("symUserNames", "B\u00fcy\u00fck m\u00fccenneb (flat)"),
-      QT_TRANSLATE_NOOP("symUserNames", "B\u00fcy\u00fck m\u00fccenneb (sharp)"),
+      //(QT_TRANSLATE_NOOP("symUserNames", "B\u00fcy\u00fck m\u00fccenneb (flat)"),
+      QT_TRANSLATE_NOOP("symUserNames", "B\u00fcy\u00fck mücenneb (flat)"),
```

Comment:
Are you sure? Afair, we replace non standard symbols with the related code, don't we?

Reply:
Not 100%,  but is seems `lubdate` ewrrnouisly takes and converts `\u00fcce` (UTF16?, shopuld do that only on `\U` IMHO), rather than just `\u00fc` (UTF8) and leaving the traling `ce` in place

Reply:
Instead of"mücenneb" this then turns into "mﳎnneb"

Reply:
It is a hack/workaround, for sure

Reply:
Now I am sure ;-) Those string now do show up correctly on Transifex